### DISCUSSION TITLE
2023/10/14/blog-history/

### DISCUSSION THREAD

**Discussion by giscus** [Andonade/Andonade.github.io#3] [Announcements]
# 2023/10/14/blog-history/

这篇文章想要记录一下我的个人博客的由来，从时间线来说，这可以追溯到大一刚开学不久，且让我从头慢慢说起 萌芽计算机的学习离不开网上大量的文档、博客，大一刚入学，需要学习许多新知识的我自然少不了接触一些博客，现在回忆起来，记忆所能探寻的最深处大抵是廖雪峰的 Git 教程，没有去参加科协基础技能培训的我就是靠着这个博客学会了 Git 的基本操作 在之后的学习生活中，开始使用 Github 的我接触到了几

https://andonade.github.io/2023/10/14/blog-history/

**Comment by Andonade** [comment]
qwq

**Comment by chillcicada** [comment]
复习累了来你这转转
